delivery_multi_destination/models/delivery_carrier.py

```diff
@@ -69,23 +70,22 @@ def rate_shipment(self, order):
         """We have to override this method for getting the proper price
         according destination on sales orders.
         """
-        if self.destination_type == "one":
-            return super().rate_shipment(order)
-        else:
+        if self.destination_type != "one":
             carrier = self.with_context(show_children_carriers=True)
             for subcarrier in carrier.child_ids:
                 if subcarrier._match_address(order.partner_shipping_id):
                     return super(
                         DeliveryCarrier,
                         subcarrier,
                     ).rate_shipment(order)
+        return super().rate_shipment(order)
```

Comment:
The error is gone indeed... but this logic looks the same to me :thinking: What am I missing? :sweat_smile:

Reply:
This is not correct. `super` is being called twice.